app/build.gradle

```diff
@@ -1,13 +1,16 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
```

Comment:
This is not related to your PR. Could you please avoid such big changes in this PR.
This comment also applies to all the library changes that you made. Unless you really need them for updating POJO classes to Kotlin, I suggest you revert it.

Reply:
Alright, I'll revert the versions. Rest of the library changes and additions are necessary to configure and use Kotlin in the project, so they are to be kept.

Reply:
Reverted back `compileSdkVersion` and `targetSdkVersion` to 28.